mammoth/models/model.py

```diff
@@ -11,7 +13,7 @@ class BaseModel(nn.Module):
     def __init__(self, encoder, decoder, attention_bridge):
         super(BaseModel, self).__init__()
 
-    def forward(self, src, tgt, lengths, bptt=False, with_align=False):
+    def forward(self, src, tgt, lengths, return_attention=False):
```

Comment:
sorry, I' a bit lost with this attention-returning schrödinger state. Is this kwarg still relevant? (same for line 30)